bin/validate.py

```diff
@@ -10,9 +10,10 @@
     submission_id = sys.argv[1]
     predictions_path = sys.argv[2]
     invalid_reasons = []
-    if predictions_path is None:
+    if "INVALID" in predictions_path:
```

Comment:
What happens if someone names their submission with "INVALID" in it?

Reply:
If someone adds it to their filename, their submission will show up as invalid and will not be scored. So while unlikely, that's an edge-case that's not being covered. To cover it, I can make this conditional also require that the content in the file is a string that starts with "Expected 1 output file in..." etc. This is the error message that gets extracted from the file and thrown back in the form of an annotation, and in the e-mail.

Reply:
Actually, after thinking about this more, I think the submission should still be marked `INVALID` if someone names their file that way. I'm going to speak with CNB to update their instructions with regards to the truncated Docker execution log + also to explicitly instruct users to only name their files `predictions.[ext]`. If a file is named anything else, it should fail, and I'll update the error message to reflect this as a possibility. In other words, the error message will be this:

_Expected 1 Docker container output file with base name "predictions" in the output directory. Got 0, or a file incorrectly named. If multiple output files are generated, please zip them into a single file for processing._